dbcon/joblist/tupleconstantstep.cpp

```diff
@@ -585,7 +587,9 @@ void TupleConstantStep::formatMiniStats()
 }
 
 // class TupleConstantOnlyStep
-TupleConstantOnlyStep::TupleConstantOnlyStep(const JobInfo& jobInfo) : TupleConstantStep(jobInfo)
+TupleConstantOnlyStep::TupleConstantOnlyStep(const JobInfo& jobInfo)
+  : TupleConstantStep(jobInfo)
+  , fEmptySet(jobInfo.constantFalse)
```

Comment:
I would call this impossible condition b/c there are a variety of reasons when empty set can delivered.

Reply:
The naming emptySet first:
- renames jobInfo.constantFalse so that we loose connection b/w jobInfo.constantFalse and new field
- emptySet doesn't tell me why exactly do we get an empty set.
What if we preserve the naming using TupleConstantOnlyStep::constantFalse ?

Reply:
This flag governs decision to output no data.

While JobInfo contains a fact that condition is constant false in an appropriately named field, this flag contains a decision to output empty set result and as such is appropriately named.